### PR TITLE
column-navigation ordering

### PR DESCRIPTION
**Pull request by marcelmoosbrugger**
__Tasks:__

- [x] refactor code
- [x] implement ordering
- [x] create demos
- [x] fixed position of info-icons
- [x] added tooltips to info-icons
- [ ] gather feedback for my changes

__Changes affecting other parts of software :__

* [Options] renamed "showEdit" to "showOptions"
* [Options] renamed "editIcon" to "actionIcon"
* [Options] renamed "showEditIcon" to "showActionIcon"
* [Event] renamed "husky.column-navigation.edit" to "husky.column-navigation.action"

### CODE REVIEW

File: husky_components/overlay/main.js [hunk 1]
```diff
@@ -463,12 +463,6 @@ define([], function() {
          * Removes the component
          */
         removeComponent: function() {
-            // todo fix bug: sometimes overlay-sandbox has own sandbox or parent-sandboxes as child which
-            // couses an endless loop. The bug can be reproduced by starting the component
-            // in a clickhandler with openOnStart-option true
-            // this.sandbox.stop();
-
-            this.sandbox.stop('*');
```
Comment (wachterjohannes) [inline husky_components/overlay/main.js]: is this bug fixed? @marcelmoosbrugger

File: husky_components/overlay/main.js [hunk 2]
```diff
@@ -989,13 +983,15 @@ define([], function() {
...
+                this.slides[this.activeSlide].cancelCallback,
+                element = null;
 
             if (!!event) {
                 this.sandbox.dom.preventDefault(event);
                 this.sandbox.dom.stopPropagation(event);
+                element = event.currentTarget;
             }
-            if (this.executeCallback(cancelCallback, event.currentTarget) !== false) {
+            if (this.executeCallback(cancelCallback, element) !== false) {
```
Comment (wachterjohannes) [inline husky_components/overlay/main.js]: made this PR obsolete https://github.com/massiveart/husky/pull/448

File: husky_extensions/confirm.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
+(function() {
```
Comment (wachterjohannes) [inline husky_extensions/confirm.js]: add header here

File: husky_extensions/confirm.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+                            }
+                        }
+                    ]);
+                }
+            };
+        }
+    });
+})();
+
+
```
Comment (wachterjohannes) [inline husky_extensions/confirm.js]: only one empty line at the end!

File: husky_extensions/confirm.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+                    that.sandbox.start([
+                        {
+                            name: 'overlay@husky',
+                            options: {
+                                el: $element,
+                                title: app.sandbox.translate(title),
+                                message: app.sandbox.translate(message),
+                                closeCallback: closeCallback,
+                                okCallback: okCallback,
+                                type: 'warning'
```
Comment (wachterjohannes) [inline husky_extensions/confirm.js]: make the type as optional parameter (default `warning`)

File: husky_components/overlay/main.js (hunk 1 above)
Comment (marcelmoosbrugger) [inline husky_components/overlay/main.js]: @wachterjohannes yes and no. it seems to work properly everywhere except in the demos. But the bug in the demos just about the overlay but about stoping components in general

File: husky_extensions/confirm.js (hunk 3 above)
Comment (marcelmoosbrugger) [inline husky_extensions/confirm.js]: @wachterjohannes done. as well as in all other extensions

File: husky_extensions/confirm.js (hunk 4 above)
Comment (marcelmoosbrugger) [inline husky_extensions/confirm.js]: @wachterjohannes done

File: husky_extensions/confirm.js (hunk 5 above)
Comment (marcelmoosbrugger) [inline husky_extensions/confirm.js]: @wachterjohannes done

File: husky_components/overlay/main.js (hunk 1 above)
Comment (wachterjohannes) [inline husky_components/overlay/main.js]: @marcelmoosbrugger you have tested it well in sulu?

**Comment by wachterjohannes** [comment]
@marcelmoosbrugger when the column navigation is in ordering mode it should not fire the event action ... thats annoying (=

**Pull request merged**